boards/BMS/Inc/App/App_BmsWorld.h

```diff
@@ -8,11 +8,8 @@
 struct BmsWorld;
 
 /**
- * Allocate and initialize a world in the application layer
- * @param can_tx_interface The CAN TX interface to register to the world
- * @param can_rx_interface The CAN RX interface to register to the world
- * @param imd The IMD to register to the world
- * @param heartbeat_monitor The heartbeat monitor to register to the world
+ * Allocate and initialize a world in the application layer by registering
```

Comment:
The javaco for world `World_Create` is getting a bit out of hand since I added like 8 new ones for PDM. Decided to just remove the `@param` altogether.